### PR TITLE
3.2 - Surf and Paddle - Andrew T. Poe

### PR DESCRIPTION
**Pull request by andrewtpoe**
Primary objectives complete. Gif tax to be paid soon.

### CODE REVIEW

**Comment by rpearce** [comment]
Think this is 3.4 not 3.2...

File: index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+          <img class="sec__authorPic" src="./images/dolphin.png">
+          <div class="sec__imgOverlay"></div>
+          <div class="sec__authorName">
+            ANNIE DOLPHIN
+          </div>
+          <div class="sec__authorSocialMedia">
+            <ul>
+              <li><a href="#">Twitter</a></li>
+              <li><a href="#">Facebook</a></li>
+              <li><a href="#">Instagram</a></li>
```
Comment (rpearce) [inline index.html:74]: Heads up to be very careful linking to a hash location. It's not going to matter here, but html5 `pushState` frameworks/libs often respond to "hash changes", so be aware of the tech involved

File: sass/_article.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.article {
+  position: relative;
+  margin: 40px 10% 0 10%;
```
Comment (rpearce) [inline sass/_article.scss:3]: you don't need the last `10%`, for it is already set by the right side. here, you are overriding `10%` with `10%`

File: sass/_article.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    color: $accentColor2;
+    &:hover {
+      color: $accentColor1;
+    }
+    &:visited {
+      color: $subTextColor;
+    }
+  }
+  > h2 {
+    margin: 30px 0px 30px 0px;
```
Comment (rpearce) [inline sass/_article.scss:39]:
``` scss
margin: 30px 0;
```

File: sass/_article.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+.article__extraNav {
+  > ul {
+    padding: 0px;
+    vertical-align: middle;
+    > li {
+      max-height: 100%;
+      width: auto;
+      margin: 0px 15px 0px 0px;
+      padding: 0px;
```
Comment (rpearce) [inline sass/_article.scss:53]: remember, measurement types not required on `0`

File: sass/_mixins.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@mixin transform($xVal, $yVal: null) {
+  -webkit-transform: translateX($xVal);
+  -webkit-transform: translateY($yVal);
+  transform: translateX($xVal);
+  transform: translateY($yVal);
```
Comment (rpearce) [inline sass/_mixins.scss:5]: this seems a bit repetitive... does this work? I don't actually know if you can pass null.

``` scss
@mixin transform($x: null, $y: null) {
  -webkit-transform: translate($x, $y);
  transform: translate($x, $y);
}
```

**Comment by rpearce** [comment]
prodigious

File: index.html (hunk 1 above)
Comment (andrewtpoe) [inline index.html:74]: Will watch out for this in the future. Here it was just used as a placeholder link.

**Pull request closed without merging**